manager/manager_test.go

```diff
@@ -18,7 +18,7 @@ import (
 func TestManager_cleanupStaleRequests(t *testing.T) {
 
 	ctx := context.TODO()
-	log := logrtesting.NewTestLogger(t)
+	log := testr.New(t)
```

Comment:
`logrtesting.NewTestLogger()` is deprecated, so change it with `testr.New()` function per suggestion.